src/boilerplates/boilerplates.ts

```diff
@@ -42,6 +43,29 @@ export function makeCommandBoilerplates(program: Command) {
         .action((options) => {
             createEmbeddedWalletReact(program, options);
         })
+    
+    comm
+        .command("create-google-embedded-wallet-react-starter")
+        .description("Clone a starter boilerplate for Google Widget Authenticated Sequence Embedded Wallet integrated with React")
+        .option(
+            "--waas-config-key <waas_key>",
+            "WaaS config key for this project"
+        )
+        .option(
+            "--project-access-key <access_key>",
+            "Project access key for Sequence requests"
+        )
+        .option(
+            "--google-client-id <google_client_id>",
+            "Google client ID to be used during authentication"
+        )
+        .option(
+            "--verbose",
+            "Show additional information in the output"
+        )
```

Comment:
Remove the indentation within each .option to maintain consistency with the other commands